CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,157 @@
+  - [Contributors](#contributors)
+  - [Reviewers](#reviewers)
+  - [Maintainers](#maintainers)
+- [Inactive members](#inactive-members)
+
+## Roles
+
+### Community participants
+
+Community participants engage with Zarf, 
+contributing their time and energy in discussions or just generally helping out.  Additionally, community members participate in [Zarf community meetings](https://github.com/zarf-dev/zarf/issues/2613).
+
+#### Responsibilities
+
+- Keep participating!
+
+#### Prerequisites
+
+- Must follow the [OpenSSF Code of Conduct]
+- Must follow the [Contribution Guide]
+
+### Contributors
+
+Contributors help advance the Zarf project through commits, issues, and pull requests.  Contributors participate through GitHub teams,
+and pre-submit tests are automatically run for their PRs.
+
+#### Responsibilities
+- Can be assigned issues and PRs
+- Responsive to issues and PRs assigned to them
+- Others can ask for reviews with a `/cc @username`.
+- Responsive to mentions of teams they are members of
+- Active owner of code they have contributed (unless ownership is explicitly transferred)
+  - Ensures code is well tested and that tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+
+#### Privileges
+
+- Tests run against their PRs automatically
+
+#### Prerequisites
+
+- Enabled two-factor authentication on their GitHub account
+- Have made contributions to the project in the form of:
+  - Authoring or reviewing PRs on GitHub. At least one PR must be **merged**.
+  - Filing or commenting on issues on GitHub
+  - Contributing to a project, or community discussions (e.g. meetings, Slack,
+    email discussion forums, Stack Overflow)
+- Active contributor to Zarf
+
+#### Promotion process `NEEDS REFINED?`
+
+- Make at least one commit to a repository's code or open a pull request that gets merged into the repository
+
+### Reviewers
+
+Reviewers are trusted members of the Zarf community that are able to review changes to Zarf and indicate if those changes are ready for merge.  Reviewers have a strong and active track record of contribution to the Zarf project.
+
+#### Responsibilities
+
+Commits to being an active contributor and reviewer as part of the Zarf project.  'Active' is defined as six PRs or PR reviews (or mix thereof) in six months.
+Is supportive of new and occasional contributors and helps get useful PRs in shape to commit.
+
+#### Additional Privileges
+
+Has GitHub or CI/CD rights to approve pull requests in specific directories
+
+#### Prerequisites
+
+Experience as a Contributor for at least 6 months
+Is an Organization Member
+Has reviewed, or helped review, at least 10 (`?`)Pull Requests
+Has analyzed and resolved test failures
+Has demonstrated an in-depth knowledge of Zarf
+
+#### Promotion process
+
+- Sponsored by a maintainer
+  - With no objections from other maintainers
+  - Done through PR to update the CODEOWNERS file, and addition to Zarf Maintainer group
```

Comment:
It may only be a change to the underlying group in the organization - but organization invite would be extended.